docs/installingLocal.md

```diff
@@ -11,31 +11,64 @@ This page describes the procedure to run PrairieLearn within Docker, but using a
 git clone https://github.com/PrairieLearn/PrairieLearn.git
 ```
 
-* Install the Node.js packages.  This will use the version of `npm` that is pre-installed in the Docker image, so you don't need your own copy installed.
+* Run PrairieLearn with:
 
 ```sh
-docker run --rm -w /PrairieLearn -v /path/to/PrairieLearn:/PrairieLearn prairielearn/prairielearn /usr/local/bin/npm ci
+docker run -it --rm -p 3000:3000 -v /path/to/PrairieLearn:/PrairieLearn prairielearn/prairielearn /bin/bash
+
+# following commands are inside the container:
+cd /PrairieLearn
+npm ci                   # install packages, repeat this after switching branches or pulling new code
+docker/start_support.sh  # start the DB, cache, etc.
+node server              # run PrairieLearn itself
+
+# now you can Ctrl-C and run "node server" again to restart PrairieLearn (after code edits, for example)
+# or Ctrl-C to stop PL and Ctrl-D to exit the container
 ```
 
-The path `/path/to/PrairieLearn` should be replaced with the *absolute* path to the PrairieLearn source on your computer.  If you're in the root of the source directory already, you can substitute `%cd%` (on Windows cmd), `${PWD}` (on Windows PowerShell), or `$PWD` (Linux, MacOS, and WSL) for `/path/to/PrairieLearn`.
+The path `/path/to/PrairieLearn` above should be replaced with the *absolute* path to the PrairieLearn source on your computer.  If you're in the root of the source directory already, you can substitute `%cd%` (on Windows cmd), `${PWD}` (on Windows PowerShell), or `$PWD` (Linux, MacOS, and WSL) for `/path/to/PrairieLearn`.
 
-By default, PrairieLearn will load `exampleCourse`, `testCourse`, and any courses mounted at `/course` and `/course[2-9]` in the Docker container.  To override this behavior, you can create a custom [`config.json` file](configJson.md).
 
-* Run it with:
+## Running the test suite
+
+The linters and tests for the Python and JavaScript code can be run with the following commands in a shell instance:
 
 ```sh
-docker run -it --rm -p 3000:3000 -v /path/to/PrairieLearn:/PrairieLearn prairielearn/prairielearn
+docker run -it --rm -p 3000:3000 -v /path/to/PrairieLearn:/PrairieLearn prairielearn/prairielearn /bin/bash
+
+# following commands are inside the container:
+cd /PrairieLearn
+./docker/lint_js.sh
+./docker/lint_python.sh
+./docker/test_js.sh
+./docker/test_python.sh
```

Comment:
Linters could also be invoked with:
```
make lint-js
make lint-python
```
I believe the `make` target is easier to memorize. It may be a good idea to create `make` targets that include the test plus initialization as well, though I don't have a strong opinion on it.

Reply:
I changed to use `make` targets and I added targets for `test-js` and `test-python`. I didn't add targets that run the initialization because that causes problems with re-running those targets. I want to keep it so that you can run `make test` or `make lint` repeatedly, as part of a debug-edit-check cycle.